src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -376,9 +376,21 @@ ReportClientSideStats(
   }
 
   std::cout << "    Request count: " << stats.request_count << std::endl;
-  if (stats.delayed_request_count != 0) {
-    std::cout << "    Delayed Request Count: " << stats.delayed_request_count
-              << std::endl;
+  float delay_pct =
+      ((float)stats.delayed_request_count / stats.request_count) * 100;
+  float delay_pct_threshold = 1.0;
```

Comment:
make this a constant somewhere, not here.

Reply:
Done. Moved to a private class variable.